README.md

```diff
@@ -76,6 +75,43 @@ You can find more options in `sqllogictest --help`.
 
 Note that only postgres is supported now.
 
+## `.slt` Test File Format Cookbook
+
+Test files have the `.slt` extension and use a dialect of Sqlite [Sqllogictest].
```

Comment:
The extension is optional. It can be anything. Some ppl prefer `.test` 🤣 
https://github.com/risinglightdb/sqllogictest-rs/issues/118